### ISSUE TITLE
A11y_Visual Studio Code_ Remove Terminal_ Keyboard: Not able to remove the Split Terminals using Keyboard.

### ISSUE THREAD

**Issue by Latha2564** [microsoft/vscode-docs#4790]
GitHubTags:#A11y_VSCode;#A11ySev2;#A11yTCS;#A11yMAS;#DesktopApp;#Visual Studio Code Client;#WCAG2.1.1;#Keyboard;#MAC;#AINotInScope;#VoiceOver;#Benchmark-VisualStudioCode-MAC-Aug2021
### Environment Details:​
Application Name: Visual Studio Code
Standard Violated: Keyboard

### Repro Steps:​
1.Open Visual Studio Code editor
2.Tab till any Program code tab and press enter.
3.Tab till Terminal and press Enter.
4.  Split the Terminals and try to delete or remove the terminals .
5. Verify whether able to delete or remove the Split Terminals using Keyboard or not.
### Actual Result:
Not able to remove the Split Terminals using Keyboard

### Expected Result:
Users should  able to remove the Split Terminals using Keyboard

### User Impact:
Keyboard Users will find difficult in deleting the split terminals if the users are not able to remove the Split Terminals using Keyboard

**Comment by Latha2564** [comment]
https://user-images.githubusercontent.com/89508906/131302485-7b6c5017-a237-4bfe-9bc8-0c6b48dcf45e.mp4

**Comment by gregvanl** [comment]
@Latha2564 Issues related the VS Code product UI should be created in https://github.com/microsoft/vscode/issues.
This repository is for documentation or code.visualstudio.com website issues.
Moving to vscode repo.
